src/utils/parser/tests/getScopes.spec.js

```diff
@@ -0,0 +1,45 @@
+import getScopes from "../getScopes";
+import { setSource } from "../sources";
+
+describe("Parser.getScopes", () => {
+  it("parses simple generated/minified es5 source", () => {
+    const source = {
```

Comment:
nit: we can use the parser's test helper and save this in its own file i think

its used via
```javascript
import { getSource } from "../helpers";

//...
const source = getSource("getScopes/source1");
```